lib/rom/relation.rb

```diff
@@ -189,6 +191,11 @@ def repository
       self.class.repository
     end
 
+    # @api public
+    def to_lazy(*args)
+      Lazy.new(self, *args)
+    end
```

Comment:
Naming convention from `Enumerable` suggests that a simple `#lazy` here would suffice.  Or at least as an alias.